imports/plugins/core/system-info/server/no-meteor/queries/systemInformation.js

```diff
@@ -0,0 +1,26 @@
+import packageJson from "/package.json";
```

Comment:
I've started thinking about how we're soon wanting to move plugins to be npm packages. An import from outside the plugin like this one won't work in that case. At the moment, we only need the version. Instead, there are two files where `new ReactionNodeApp()` is done. You can import `packageJson` in those two files, and add the version to context in the `ReactionNodeApp` constructor options: `appVersion: packageJson.version`.

Then here you can use `context.appVersion`.